tools/Makefile

```diff
@@ -42,8 +42,10 @@ update-version:
 	## When manually updating major version
 	./releaser/scripts/update-version.sh
 
-.PHONY: generate_mocks
-generate_mocks:	
-	go install github.com/vektra/mockery/v2@v2.42.1
+.PHONY: generate_mocks_linux
+generate_mocks_linux:	
 	rm -Rf ../mockadmin
-	mockery
+	curl -L -o mockery.tar.gz https://github.com/vektra/mockery/releases/download/v2.42.1/mockery_2.42.1_Linux_x86_64.tar.gz
```

Comment:
From [this comment](https://github.com/mongodb/atlas-sdk-go/pull/287#discussion_r1537256974) in the previous PR, this dependency would not be automatically updated, right?

Reply:
not sure I understand your question

Reply:
Sorry, I meant that as far as I understand, @wtrocki wanted the mockery dependency version to be elsewhere so that it can be automatically updated by dependabot. As it is now, I don't think that's the case

Reply:
Related to the https://github.com/mongodb/atlas-sdk-go/pull/288#discussion_r1537657532

Re version - I think we use pattern to define them in the header of the markdown to understand how to update them and cover them in the Contributing guide